test/e2e/tests/account/unlock-wallet.spec.ts

```diff
@@ -38,6 +38,8 @@ describe('Unlock wallet - ', function () {
         await loginPage.checkIncorrectPasswordMessageIsDisplayed();
         await loginPage.loginToHomepage();
         await homePage.checkPageIsLoaded();
+        // Wait for balance as the test can still fail if the process is terminated too early
+        await homePage.checkExpectedTokenBalanceIsDisplayed('25', 'ETH');
```

Comment:
Here's one of the runs that showed the issue. It's very recurrent.

https://github.com/MetaMask/metamask-extension/actions/runs/18967553826/job/54168354578?pr=36808

With this PR changes, we are loading the UI earlier as we no longer artificially wait for AccountTrackerController to finish syncing, so this was probably an unintended consequence.